### PR TITLE
Add Python Coincurve Library

### PR DESCRIPTION
**Pull request by OverlordQ**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ X GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ofek** [comment]
Any update?

File: recipes/coincurve/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - requests
```
Comment (scopatz) [inline recipes/coincurve/meta.yaml:20]: You don't need requests in build

File: recipes/coincurve/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - pkg-config
+  host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (scopatz) [inline recipes/coincurve/meta.yaml:24]: Please remove compilers from host

File: recipes/coincurve/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - coincurve
+
+about:
+  home: "https://github.com/ofek/coincurve"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "Cross-platform Python CFFI bindings for libsecp256k1"
+  doc_url:
+  dev_url:
```
Comment (scopatz) [inline recipes/coincurve/meta.yaml:46]: please remove empty entries

**Comment by OverlordQ** [comment]
I asked on Gitter but never got a response, but the only way I could get it to build was with the compilers on host, other wise you get this: 

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=184307&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=881

So if there's a fix for that, I'd love to know it.

**Comment by scopatz** [comment]
That seems like this is an issue with the package itself

**Comment by ofek** [comment]
Is this what you need? https://github.com/ofek/coincurve/pull/35

**Pull request closed without merging**